.github/workflows/build_all_packages.yml

```
@@ -0,0 +1,62 @@
+name: Build All Packages
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install prerequisite packages
+        run: |
+          sudo apt-get install \
```

How about

```suggestion
          sudo apt install -y \
```

just for consistency with the line

```
sudo apt install -y pciutils zip libncurses5 python libpython2.7
```